user/pages/03.commerce2/01.user-guide/10.commerce-pos/docs.md

```diff
@@ -0,0 +1,15 @@
+---
+title: POS
+taxonomy:
+    category: docs
+---
+
+! We need help filling out this section! Feel free to follow the *edit this page* link and contribute.
+
+POS is enabled in Drupal Commerce 2.x with an external module
```

Comment:
I think better to say "Commerce POS (Point of Sale) is a Drupal module built on top of the Drupal Commerce installation.